src/components/Tabs/TabList/__tests__/__snapshots__/tabList-snapshot-tests.jest.js.snap

```diff
@@ -2,21 +2,23 @@
 
 exports[`TabList renders correctly stretched tab list 1`] = `
 <div
-  className="tabs--wrapper stretched"
+  className="tabsWrapper tabs--wrapper stretched stretched"
+  data-testid="tab-list"
 >
   <ul
-    className="tabs-list"
+    className="tabsList tabs-list undefined"
```

Comment:
I think we talked about this bug before? is it the one that related to the get style or?

Reply:
approved the other pr with the fixed :)